pkg/skaffold/runner/runner_test.go

```diff
@@ -321,3 +322,24 @@ func TestRun(t *testing.T) {
 		})
 	}
 }
+
+func TestGetContextEnvironment(t *testing.T) {
+	os.Setenv("KUBE_CONTEXT", "test-environment-context")
+	defer os.Unsetenv("KUBE_CONTEXT")
+
+	if c := getContext("test-kube-context", ""); c != "test-environment-context" {
```

Comment:
This test should make clear that the environment overrides the context from the schema. For that, the schema-context cannot be `""`.